src/DSFML/Audio/SoundBuffer.cpp

```diff
@@ -68,17 +68,17 @@ void sfSoundBuffer_destroy(sfSoundBuffer* soundBuffer)
     delete soundBuffer;
 }
 
-DBool sfSoundBuffer_saveToFile(const sfSoundBuffer* soundBuffer, const char* filename)
+DBool sfSoundBuffer_saveToFile(const sfSoundBuffer* soundBuffer, const char* filename, size_t length)
 {
-    return (soundBuffer->This.saveToFile(filename))? DTrue: DFalse;
+    return (soundBuffer->This.saveToFile(std::string(filename, length)))? DTrue: DFalse;
 }
 
 const DShort* sfSoundBuffer_getSamples(const sfSoundBuffer* soundBuffer)
 {
     return soundBuffer->This.getSamples();
 }
 
-size_t sfSoundBuffer_getSampleCount(const sfSoundBuffer* soundBuffer)
```

Comment:
Any reason for this change? We might need a cast on the return statement, but I think it should still return a size_t.

sfSoundBuffer_getSampleCount is defined as returning a size_t on the D side.